flow/connectors/postgres/qvalue_convert.go

```diff
@@ -391,9 +391,29 @@ func convertToArray[T any](kind types.QValueKind, value any) ([]T, error) {
 }
 
 func (c *PostgresConnector) parseFieldFromPostgresOID(
-	oid uint32, typmod int32, value any, customTypeMapping map[uint32]shared.CustomDataType, version uint32,
+	oid uint32,
+	typmod int32,
+	value any,
+	customTypeMapping map[uint32]shared.CustomDataType,
+	version uint32,
+) (types.QValue, error) {
+	return c.parseFieldFromPostgresOIDWithQValueKind(
+		oid,
+		typmod,
+		c.postgresOIDToQValueKind(oid, customTypeMapping, version),
+		true,
```

Comment:
this matches previous behavior, but is incorrect: we should be maintaining correct null info during CDC